public/pages/games/mistureba.html

```diff
@@ -6,122 +7,241 @@
+                <path d="M10.121 12.596A6.48 6.48 0 0 0 12.025 8a6.48 6.48 0 0 0-1.904-4.596l-.707.707A5.483 5.483 0 0 1 11.025 8a5.483 5.483 0 0 1-1.61 3.89l.706.706z"/>
+                <path d="M10.025 8a4.486 4.486 0 0 1-1.318 3.182L8 10.475A3.489 3.489 0 0 0 9.025 8c0-.966-.392-1.841-1.025-2.475l.707-.707A4.486 4.486 0 0 1 10.025 8zM7 4a.5.5 0 0 0-.812-.39L3.825 5.5H1.5A.5.5 0 0 0 1 6v4a.5.5 0 0 0 .5.5h2.325l2.363 1.89A.5.5 0 0 0 7 12V4zM4.312 6.39 6 5.04v5.92L4.312 9.61A.5.5 0 0 0 4 9.5H2v-3h2a.5.5 0 0 0 .312-.11z"/>
+              </svg>
+            </button>
+            <button type="button"  class="btn btn-outline-success" onclick="tip()" >
+             <strong>	 
+               OK
+             </strong>
+           </button>
+          </div>
+        </div> 
+      </div>   
+    </aside>
+  </main>
+ </div> 
+  
 
-      </div>
-      <hr>
-    </div>
-    <div class="result1">
-      <div class="result">
-        <h4>PONTOS TOTAIS</h4>
-        <h4>0</h4>
 
-      </div>
 
-    </div>
-    <hr>
-  </main>
 
-  <!---Começo Rodapé-->
-  <footer class="bg-light text-center text-lg-start">
-    <!-- Copyright -->
-    <div class="text-center p-3" style="background-color: rgba(0, 0, 0, 0.2)">
-      © 2022 Copyright:
-      <a class="text-dark" href="#">RecodePro 2022 - Módulo 2 Atividade prática Squad 61.</a>
+<!---INICIO RODAPÉ-->
+<footer class=" footer-container py-5 conteiner-fluid row">
+  <div class="col-6 col-md">
+    <img width= "200px + 10vw" src="../../../public/assets/images/logo.png">
+  </div>
+  <div class="col-6 col-md">
+        <h5 class="title text-secondary">
+          Termos:
+        </h5>
+      <ul class="list-unstyled text-small">
+        <li><a class="text-muted" href="https://recodepro.org.br/" target="_blank">Recode Pro</a></li>
+        <li><a class="text-muted" href="" target="_blank">Termos e Condições</a></li>
+        <li><a class="text-muted" href=""target="_blank">Política de Privacidade</a></li>
+      </ul>
+    </div>
+    <div class="col-6 col-md">
+        <h5 class="title text-secondary">
+          Canais de Atendimento:
+        </h5>
+        <p>De segunda á sexta-feira: 08:00 ás 20:00</p>
+        <img class="img-fluid" src="../../../public/assets/images/contato.png"> +55 00 0000-0000 </a>
+        <p><img src="../../../public/assets/images/email.png"> recreio.org@gmail.com </a> </p>
     </div>
-    <!-- Copyright -->
+    <div class="col-6 col-md">
+        <h5  class="title text-secondary">
+          Follow us:
+        </h5>
+        <ul class="list-unstyled text-small">
+        <a href="https://github.com/Squad-61/" target="_blank"> <img src="../../../public/assets/images/github.png"> </a>
+        <a href="https://www.instagram.com/" target="_blank"> <img src="../../../public/assets/images/instagram.png"> </a>
```

Comment:
Sugestão de correção:

```suggestion
        <a href="https://www.instagram.com/" target="_blank"> <img src="../../assets/images/instagram.png"> </a>
```